entrypoint.sh

```diff
@@ -78,52 +73,40 @@ elif [[ "${black_exit_val}" -eq "1" && "${reviewdog_exit_val}" -eq "1" ]]; then
+  echo "[action-black] ERROR: Black found a syntax error when checking the" \
+    "files (error code: ${black_exit_val})."
+elif [[ "${black_exit_val}" -eq "123" && "${reviewdog_exit_val}" -eq "0" ]]; then
+  black_error="true"
+  reviewdog_error="false"
+  echo "[action-black] ERROR: Black found a syntax error when checking the" \
+    "files (error code: ${black_exit_val})."
 else
-  if [[ "${black_exit_val}" -ne "0" && "${black_exit_val}" -ne "1" && \
+  if [[ "${black_exit_val}" -eq "123" && "${reviewdog_exit_val}" -ne "0" && \
+    "${reviewdog_exit_val}" -ne "1" ]]; then
+    echo "[action-black] ERROR: Black found a syntax error when checking the" \
+      "files (error code: ${black_exit_val})."
+    echo "[action-black] ERROR: Something went wrong while trying to run the" \
+      "reviewdog error annotator (error code: ${reviewdog_exit_val})."
+  elif [[ "${black_exit_val}" -ne "0" && "${black_exit_val}" -ne "1" && \
     "${reviewdog_exit_val}" -ne "0" && "${reviewdog_exit_val}" -ne "1" ]]; then
     echo "[action-black] ERROR: Something went wrong while trying to run the black" \
       "formatter while annotating the changes using reviewdog (black error code:" \
       "${black_exit_val}, reviewdog error code: ${reviewdog_exit_val})."
-    exit 1
   elif [[ "${black_exit_val}" -ne "0" && "${black_exit_val}" -ne "1" ]]; then
     echo "[action-black] ERROR: Something went wrong while trying to run the black" \
       "formatter (error code: ${black_exit_val})."
-    exit 1
   else
     echo "[action-black] ERROR: Something went wrong while trying to run the" \
       "reviewdog error annotator (error code: ${reviewdog_exit_val})."
-    exit 1
   fi
-fi
-
-# Also format code if this is requested
-# NOTE: Useful for writing back changes or creating a pull request.
-black_format_exit_val="0"
-if [[ "${INPUT_FORMAT,,}" = 'true' && "${black_error}" = 'true' ]]; then
-  echo "[action-black] Formatting python code using the black formatter..."
-  black ${black_args[@]} || black_format_exit_val="$?"
-
-  # Check whether black formatting was succesfull
-  if [[ "${black_format_exit_val}" -eq "0" ]]; then
-    echo "::set-output name=is_formatted::true"
-  elif [[ "${black_format_exit_val}" -eq "1" ]]; then
-    black_error="true"
-    echo "::set-output name=is_formatted::false"
-  else
-    echo "[action-black] ERROR: Something went wrong while trying to run the" \
-      "black formatter (error code: ${black_format_exit_val})."
-    exit 1
-  fi
-elif [[ "${INPUT_FORMAT,,}" = 'true' && "${black_error}" = 'false' ]]; then
-  echo "[action-black] Formatting not needed."
-  echo "::set-output name=is_formatted::false"
-else
-  echo "::set-output name=is_formatted::false"
+  exit 1
 fi
 
 # Throw error if an error occurred and fail_on_error is true
-if [[ "${INPUT_FAIL_ON_ERROR,,}" = 'true' && (\
-  "${INPUT_ANNOTATE,,}" = 'false' && "${black_error}" = 'true' || \
+if [[ "${INPUT_FAIL_ON_ERROR,,}" = 'true' && ("${black_error}" = 'true' || \
```

Comment:
`,,`?

Reply:
@haya14busa It's bash 4.0 syntax for making strings lowercase (see [this StackOverflow post](https://stackoverflow.com/questions/2264428/how-to-convert-a-string-to-lower-case-in-bash)). I will add a comment such that it doesn't confuse other developers.